### PR TITLE
Added vertical-align to align select and textbox

### PR DESCRIPTION
**Pull request by agliga**
## Description
Looks like there is a weird chrome bug. Part of the issue got introduced when we switched to Ariel font. Seems like a rounding problem.
In order to fix this there needs to be an "odd" `line-height`. This makes it round correctly. 
Added line height of `1.5` in order to achieve this. Added a message about this as well.
Tested in firefox and chrome and works fine. Saw safari is aligned but select looks off (at least on my version)

#1253

### CODE REVIEW

File: dist/select/ds4/select.css [hunk 1]
```diff
@@ -42,6 +42,7 @@ span.select {
 .select--large select {
```
Comment (ianmcburnie) [inline dist/select/ds4/select.css:42]: Is the fix only need for the large size?

Comment (agliga) [reply dist/select/ds4/select.css:42]: I didnt see a problem with alignment on normal sizes. So put it only for large.

**Comment by agliga** [comment]
Screen Shot 2020-10-09 at 10 40 26 AM
Safari after change

**Review by ianmcburnie — APPROVED** [review]
Looking good now!

**Pull request merged**